web/tasks.go

```diff
@@ -0,0 +1,66 @@
+package web
+
+import (
+	"context"
+	"errors"
+	"strings"
+
+	"github.com/google/ts-bridge/stackdriver"
+	"github.com/google/ts-bridge/tsbridge"
+)
+
+// Sync updates all configured metrics.
```

Comment:
I am not sure Sync and Cleanup belong to the `web` package.

Reply:
Moved into a `tasks` package